spotify_confidence/analysis/frequentist/confidence_computers/bootstrap_computer.py

```diff
@@ -6,38 +6,38 @@
 from spotify_confidence.analysis.constants import CI_LOWER, CI_UPPER, SFX1, SFX2, BOOTSTRAPS, INTERVAL_SIZE
 
 
-def point_estimate(df: DataFrame, arg_dict: Dict[str, str]) -> float:
-    bootstrap_samples = arg_dict[BOOTSTRAPS]
+def point_estimate(df: DataFrame, **kwargs: Dict[str, str]) -> float:
+    bootstrap_samples = kwargs[BOOTSTRAPS]
```

Comment:
I switched to using the more pythonic **kwargs rather than passing in a dict the way we did before